.nvmrc

```diff
@@ -1 +1 @@
-v12
+v20
```

Comment:
this might be a compatibility issue since this is pr is a feature, hence it will bump minor version.
there are packages using this package with a caret version, causing them to pull this feature. forcing nvm to use v20 might break them. i would suggest to check this behavior